kong/cmd/migrations.lua

```diff
@@ -90,7 +90,7 @@ local function execute(args)
 
   package.path = conf.lua_package_path .. ";" .. package.path
 
-  conf.pg_timeout = args.db_timeout -- connect + send + read
+  conf.pg_timeout = args.db_timeout or conf.pg_timeout -- connect + send + read
```

Comment:
For simplicity, I think we can safely deprecate db_timeout, it appears that it doesn't have any practical necessity.

Reply:
Uh, I think it's useful to quickly test for the appropriate timeout value without having to change `kong.conf` or the `KONG_PG_TIMEOUT` environment variable every time.

Reply:
Also, if we deprecated this argument directly, it may cause a break change if there are users with this CLI argument. I think we should carefully consider it.

Reply:
So Kong will still prioritize `db_timeout` over `pg_timeout` when starting via kong cli, and the difference between with and without this change is that `pg_timeout` is optional or mandatory.

Reply:
The fix is to change `--db-timeout` to an optional argument and no longer set a default value for it. This way, if the user does not explicitly specify this argument, `pg_timeout` will use the value from `kong.conf`.

Note: The `pg_timeout` in `kong.conf` still has a default value.